docs/source/usage/config_files.rst

```diff
@@ -0,0 +1,190 @@
+Configuration Files
+===================
+
+Configuration files are designed to be used in conjunction with the :doc:`/usage/cli` (though they can be used with the Python API too) to help specify distribution behavior when creating generative metadata. 
+
+In the configuration file, you can specify the variable specification, distribution providers, privacy level and number of rows of data to generate. 
+
+The configuration file must be in the `.toml` format. 
+
+.. admonition:: What is TOML?
+
+   TOML, short for Tom's Obvious, Minimal Language, aims to be a minimal configuration file with easy readability. 
+
+   A TOML file consists of key/value pairs, tables (groups of key/value pairs), and optionally, arrays of tables. Values can be strings, numbers, booleans, dates/times, or arrays.
+   
+   TOML is **case sensitive**, but does not care about indentation or whitespace.
+
+   Basic TOML syntax is as follows:
+
+   .. code-block:: toml
+
+      # This is a comment
+      key = "value"  # You can comment on the same line too
+      key2 = """
+      This is a multiline string
+      """
+      answer = 42
+      bool = true # note how booleans are lowercase, as opposed to Python's 'True'/'False'
+      date = 1979-05-27T07:32:00Z
+
+
+   Tables (denoted by headers in square brackets) can be used to group key/value pairs together.
+
+   .. code-block:: toml
+
+      [table]
+      key = "value"
+
+
+   Tables themselves can be nested using dotted keys:
+
+   .. code-block:: toml
+
+      [table.subtable]
+      key = "value"
+
+   For short nested key/values, inline tables can be used:
+
+   .. code-block:: toml
+
+      inline = { key = "Foo", key2 = "Bar" }
+
+   Arrays of tables can be created by using double square brackets:
+
+   .. code-block:: toml  
+
+      [[array]]
+      key = "value"
+
+
+   For more detailed information, refer to the `TOML documentation <https://toml.io/en/>`_.
+
+
+Configuration File Structure
+----------------------------
+
+
+Specifying rows
+^^^^^^^^^^^^^^^
+
+To specify the number of rows to generate, use the ``n_rows`` key.
+
+For example:
+
+.. code-block:: toml
+
+   n_rows = 100
+
+Variable specification
+^^^^^^^^^^^^^^^^^^^^^^^
+
+To set the variable spec for a variable, create an entry in the ``[[var]]`` array. Each entry should include the following:
+
+#. ``name``: The name of the variable.
+#. ``description`` (optional): A description of the variable.
+#. ``distribution``: The distribution specification for the variable. This is a dictionary that includes the ``implements`` key specifying the distribution type, and optionally a ``parameters`` key specifying parameters for the distribution. To find distributions, and their paramters, refer to the :doc:`/api/metasyn.distribution` page.
+#. ``privacy`` (optional): The privacy specification for the variable. This is a dictionary that includes the ``name`` of the privacy, and a ``parameters`` key for specifying its paramteres.
```

Comment:
```suggestion
#. ``privacy`` (optional): The privacy specification for the variable. This is a dictionary that includes the ``name`` of the privacy, and a ``parameters`` key for specifying its parameters.
```

Reply:
Welp apparently I am incapable of spelling the word parameters.